pymc3/tuning/starting.py

```diff
@@ -171,14 +196,14 @@ def __init__(self, maxeval=5000, progressbar=True, logp_func=None, dlogp_func=No
         self.logp_func = logp_func
         if dlogp_func is None:
             self.use_gradient = False
-            self.desc = 'logp = {:,.5g}'
+            self.desc = "logp = {:,.5g}"
         else:
             self.dlogp_func = dlogp_func
             self.use_gradient = True
-            self.desc = 'logp = {:,.5g}, ||grad|| = {:,.5g}'
+            self.desc = "logp = {:,.5g}, ||grad|| = {:,.5g}"
         self.previous_x = None
-        self.progress = tqdm(total=maxeval, disable=not progressbar)
-        self.progress.n = 0
+        self.progress = progress_bar(total=maxeval, display=progressbar)
```

Comment:
This need an iterator as first argument, maybe `range(maxeval)` will do the trick